modules/reindex/src/yamlRestTest/resources/rest-api-spec/test/reindex/20_validation.yml

```diff
@@ -308,7 +308,7 @@
 ---
 "unwhitelisted remote host fails":
   - do:
-      catch: /\[badremote:9200\] not whitelisted in reindex.remote.whitelist/
+      catch: /\[badremote:9200\] not whitelisted in reindex.remote.allowlist/
```

Comment:
This will need to be updated if you change the exception message, I think.

Reply:
Same as above. (Since PR #2178 is merged, the "whitelisted" in the exception messages have changed to "allowlisted") 😁